HOWTO-HACK.txt

```diff
@@ -2,54 +2,74 @@ Instructions for folks who want to contribute to processing.py itself.
 
 Prerequisites:
 
-	git - http://git-scm.com/downloads
-	The latest JDK - http://www.oracle.com/technetwork/java/javase/downloads/index.html
-	Apache ant - http://ant.apache.org/bindownload.cgi
+    git - http://git-scm.com/downloads
+    JDK 8u144 - http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
+    Apache ant - http://ant.apache.org/bindownload.cgi
+
+JDK 8u144 is required until such time as Processing is built with JDK 9 or later.
```

Comment:
This line is brittle; their specific "best version" changes frequently with the JDK release.